flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -915,29 +936,21 @@ public void heartbeatFromResourceManager(ResourceID resourceID) {
 			default:
 				filePath = null;
 		}
+		return requestFileUploadByFilePath(filePath, timeout, fileType.toString());
+	}
 
-		if (filePath != null && !filePath.isEmpty()) {
-			final File file = new File(filePath);
-
-			if (file.exists()) {
-				final TransientBlobCache transientBlobService = blobCacheService.getTransientBlobService();
-				final TransientBlobKey transientBlobKey;
-				try (FileInputStream fileInputStream = new FileInputStream(file)) {
-					transientBlobKey = transientBlobService.putTransient(fileInputStream);
-				} catch (IOException e) {
-					log.debug("Could not upload file {}.", fileType, e);
-					return FutureUtils.completedExceptionally(new FlinkException("Could not upload file " + fileType + '.', e));
-				}
+	@Override
+	public CompletableFuture<TransientBlobKey> requestFileUploadByName(String fileName, Time timeout) {
+		log.debug("Request file name is {} upload.", fileName);
```

Comment:
Log message could be clearer. How about

```
log.debug("Received file upload request for file {}", fileName);
```
Also, I think it's enough to log once in `requestFileUploadByFilePath()`.

Reply:
Or  log.debug("Request file named {} upload.", fileName); ?

Reply:
We could add it in `requestFileUploadByFilePath().`
```code
log.debug("Received file upload request for file {}", fileTag);
```

Reply:
ok

Reply:
> We could add it in `requestFileUploadByFilePath().`
> 
> ```
> log.debug("Received file upload request for file {}", fileTag);
> ```

ok